src/popup/pages/PrivacyPolicy.vue

```diff
@@ -1,351 +1,440 @@
 <template>
   <div class="privacy-policy">
-    <p><em>The present Privacy Policy is effective since on 9th April 2020.</em></p>
-    <ol class="policy-list">
-      <li class="first">
-        <h3><strong>Introduction</strong></h3>
+    <Scrollable scrollbar-width="auto">
```

Comment:
I strongly recommend to not use the `Scrollable` component for the purpose of making the page scroll visible. The default scrollable area of the app is the `app-inner` located in App.vue.

In my opinion the page scroll should be controlled by the route meta settings as discussed on one of the meetings.

Using custom scrollable area will allow to scroll part of the screen, but the content would not be scrolled under the header, which soon will become semi-transparent.